MesaModel/hamilton/reframe_parameterisation_infrastructure/reframe_tests/parameterisation_tests.py

```diff
@@ -6,7 +6,17 @@
 
 with open("../../lhs_sampling/lhs_params.csv", "r") as f:
     csv_reader = csv.reader(f)
-    ROWS = [",".join(row[1:]) for row in list(csv_reader)]
+    ROWS = []
+    TEST_IDS = []
+    id = 0
+    for row in list(csv_reader):
+        if row[0] == str(id) or row[0] == "test_id":
```

Comment:
Does this change the behaviour from the last commit (I think that it will now add the header row to `ROWS`)? Is that intentional?

Reply:
There is no change - previously we added the header too. The tests are then parameterised with test_ids that start at 1, meaning when we index ROWS we always skip the header.

Reply:
Sorry, had misread the `1:` indexing as being on the list of rows rather than columns in the previous version.